README.md

```diff
@@ -1 +1,2054 @@
-# mailchimp-package
\ No newline at end of file
+---
+title: MailChimp package
+keywords: 
+last_updated: April 27, 2023
+tags: []
+summary: "Detailed description of the API of the MailChimp package."
+---
```

Comment:
Previously we had this readme in documentation format (with jekill) let's convert this table to html format, since it is a format accepted by the readme viewer of the Package Marketplace. (example of table in html https://raw.githubusercontent.com/slingr-stack/pandadoc-package/master/README.md )